media/CircleGraph/index.js

```diff
@@ -52,6 +52,7 @@ const viewMode = {
   viewer: 0, // default circle viewer
   selector: 1, // circle partition editor node selector
   visq: 2, // quantization error viewer
+  visqselector: 3, // quantization error viewer which is able to select nodes
```

Comment:
plz refer https://github.com/Samsung/ONE-vscode/pull/1546#discussion_r1181994078

Reply:
plz move this line below 1350 link comment

Reply:
Ahhh. Sorry.